impf/alert.py

```diff
@@ -16,6 +16,28 @@
 logger = logging.getLogger(__name__)
 p = re.compile(r"sms:\d{3}-?\d{3}")
 
+if settings.TELEGRAM_ENABLED:
```

Comment:
We can do it like this, registering a bot daemon always listening in the background. Just worried it might add an unexpected error level - especially when using concurrency/threading. I think the `telegram_send()` and `telegram_read()` functions should be sufficient for our needs, as we don't necessarily need an interactive bot - if the library allows for it. What are your thoughts?

Reply:
Bot deamon is only used during the setup phase to get the chatId automatically. After that, polling is not used and we use get_Updates() instead where we ignore all old messages with via remembering the old offset in the file. Otherwise we would get old messages from telegram.

Reply:
Do you think the current `chat_id` acquisition method is too cumbersome? Alternatively I would suggest we put this function into another module – e.g. `wizard` guiding the user through setting up alerts to keep the function interfaces clean. What do you think?